packages/components/src/Calendar/baseCalendarClassNames.ts

```diff
@@ -1,22 +1,20 @@
 /* eslint-disable camelcase */
-import { type ClassNames } from 'react-day-picker'
+import { type PropsBase } from 'react-day-picker'
 import styles from './baseCalendarClassNames.module.scss'
 
-export const baseCalendarClassNames: ClassNames = {
-  vhidden: styles.vHidden,
+export const baseCalendarClassNames: PropsBase['classNames'] = {
```

Comment:
I wonder if this is the cause of some of the DateRangerPicker style issues since this is shared in both 🤔 oddly enough it looks like the button next doesn't appear to get any of the classes applied to it, where as previous does?
<img width="542" alt="Screenshot 2025-05-08 at 3 27 48 pm" src="https://github.com/user-attachments/assets/3d256ae3-de7f-47ed-818b-017edbf2ba0b" />

Reply:
ooo dug in a little more and think the issue lies in the `LegacyCalendarRange` classnames  :) We probably just need to update these to the new namespace
![Screenshot 2025-05-08 at 3 43 13 pm](https://github.com/user-attachments/assets/51b7a2d6-e753-4b9c-a125-ab085552ea9f)